pkg/ebpf/bpf_client.go

```diff
@@ -734,8 +734,9 @@ func (l *bpfClient) computeMapEntriesFromEndpointRules(firewallRules []EbpfFirew
 		if !strings.Contains(string(firewallRule.IPCidr), "/") {
 			firewallRule.IPCidr += v1alpha1.NetworkAddress(l.hostMask)
 		}
-		//TODO - Just Purge both the entries and avoid these calls for every CIDR
-		if utils.IsCatchAllIPEntry(string(firewallRule.IPCidr)) {
+
+		if utils.IsCatchAllIPEntry(string(firewallRule.IPCidr)) ||
```

Comment:
I have removed this check as part of https://github.com/aws/aws-network-policy-agent/pull/58 but once we merge this PR, I will just keep the isNodeIP check..